.github/workflows/release-python.yml

```diff
@@ -23,7 +23,9 @@ env:
   SILK_ASSET_GROUP: django-mongodb-backend-main
   EVERGREEN_PROJECT: django-mongodb-backend
   # Constant
-  DRY_RUN: ${{ github.event_name == 'workflow_dispatch' && inputs.dry_run || 'true' }}
+  # inputs will be empty on a scheduled run.  so, we only set dry_run
+  # to 'false' when the input is set to 'false'.
+  DRY_RUN: ${{ ! contains(inputs.dry_run, 'false') }}
```

Comment:
why does it have to be a contains?
Would an equality not work the same?

Reply:
Perhaps, but this variant has already been tested. ;)